base/abstractarraymath.jl

```diff
@@ -178,7 +178,7 @@ function reverse(A::AbstractArray; dims::Integer)
 end
 
 function circshift(a::AbstractArray, shiftamt::Real)
-    circshift!(similar(a), a, (Integer(shiftamt),))
+    return circshift!(similar(a), a, (Integer(shiftamt),))
```

Comment:
Could this also be a one-line definition, like the one two lines below? Similarly with line 242.

Reply:
That would be 94 characters. which is over the 92 character limit

Reply:
Is adding `return` in long-form function definitions an official guideline for contributing to Julia? Otherwise, I wouldn't personally support changing already existing definitions like that, nor requesting contributors to follow this style. For some people used to functional style, it looks like artificially giving an imperative feel to an otherwise fine and self-sufficient expression.

Reply:
I feel like I have been told before to add it when making PRs to julia.
But I don't see it in the style-guide https://docs.julialang.org/en/v1/manual/style-guide/index.html

Might have been another repo

Reply:
```suggestion
    circshift!(similar(a), a, (Integer(shiftamt),))
```

Reply:
removed